.kokoro/build-with-run.sh

```diff
@@ -23,22 +23,23 @@ _run_error_log() {
 trap '_run_error_log' ERR
 
 # Activate mocha config
-export MOCHA_REPORTER_OUTPUT=${PROJECT}_sponge_log.xml
-export MOCHA_REPORTER=xunit
-pushd github/nodejs-docs-samples
-mv .kokoro/.mocharc.js .
-popd
+# export MOCHA_REPORTER_OUTPUT=${PROJECT}_sponge_log.xml
```

Comment:
Do we know if any internal tools depend on this data?

Reply:
Yes, they do! I didn't intend to commit those comments.